src/actions/transformMatch.js

```diff
@@ -227,6 +227,8 @@ function transformMatch(m) {
       });
       newPlayer.abilities = arr;
     }
+    newPlayer.hero_name = heroes[player.hero_id].name;
```

Comment:
make sure heroes[player.hero_id] exists, otherwise this will fail
```suggestion
    newPlayer.hero_name = heroes[player.hero_id] && heroes[player.hero_id].name;
```